text/0000-lifecycle-all.md

```diff
@@ -0,0 +1,110 @@
+# Meta
+[meta]: #meta
+- Name: Lifecycle All Binary
+- Start Date: 2020-01-21
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+### `/cnb/lifecycle/all`
+A new lifecycle binary with the name `lifecycle/all` will be included in each released lifecycle archive. Within a builder image, it will be found at the path `/cnb/lifecycle/all`. When invoked, it will run the following phases sequentially:
```

Comment:
RE: name

Creating a thread to discuss the name of this new binary.

Previous comments:

> If bin/develop gets built and get it's own binary at /cnb/lifecycle/develop should we consider /cnb/lifecycle/build or another name? 
https://github.com/buildpacks/rfcs/pull/46#issuecomment-577376828 - @jabrown85 

> build seems too close to builder and therefore likely to be confusing.
> /cnb/lifecycle/combo?
> https://github.com/buildpacks/rfcs/pull/46#issuecomment-577857191 - @ekcasey

Reply:
Not a fan of `combo`.... 

Also, FWIW, it already isn't all in that it doesn't include `rebaser`.

Reply:
@jromero gonna start a thread just to shoot down every idea without adding a suggestion? ;)

Reply:
`lifecycle/make` or `lifecycle/constructor`?

Reply:
ooo, I think `/lifecycle/make` is my favorite so far (@ashwin-venkatesh agrees 💯 )

Reply:
@ekcasey :D Wanted to start the thread but didn't have any _good_ ideas myself.

Binary name suggestions:

- `/lifecycle/phases`

Maybe we don't need a new binary at all and only provide a subcommand to the [single lifecycle binary](https://github.com/ashwin-venkatesh/rfcs/blob/multicall-binary/text/0000-lifecycle-multicall-binary-for-build.md).

- `lifecycle/lifecycle build-app`
- `lifecycle/lifecycle run-phases`

Reply:
I'm ok with `lifecycle/make` but it could be confused with [`make`](https://www.gnu.org/software/make/)